roles/collect/defaults/main.yml

```diff
@@ -1,15 +1,17 @@
 ---
 # defaults file for collect
 collect_format: 'csv'
-collect_report_name: 'pod-cpu-usage'
+collect_report_name: 'hccm-openshift-usage'
 collect_endpoint_params: '?name={{ collect_report_name }}&format={{ collect_format }}'
-collect_archive_format: xz
+collect_archive_format: gz
 collect_archive_path: '/tmp'
 collect_archive_name: korekuta
 collect_delete_after: 'true'
 collect_csv_uuid: '{{ 99999999999999999999 | random | to_uuid }}'
 collect_insights_client_cmd: '/usr/bin/insights-client'
 collect_download_path: '/tmp/korekuta-collect'
 collect_ocp_config_file: '{{ ansible_env.HOME }}/.config/ocp_usage/config.json'
-collect_oc_command: '/usr/bin/oc'
+collect_oc_command: 'oc'
```

Comment:
The above path wasn't where it was installed on the system we were using so the tasks failed.
```
$ which oc
/usr/local/bin/oc
```
Instead of changing the above to be this value I actually had the reverse thought. But we can certainly change it in the future.